theano/sandbox/cuda/opt.py

```diff
@@ -314,7 +314,7 @@ def local_gpu_split(node):
             split_node = host_input.owner
             new_op = GpuSplit(split_node.op.len_splits)
             return [new_op(gpu_from_host(split_node.inputs[0]),
-                           *split_node.inputs[1:])[host_input.index]]
+                           *split_node.inputs[1:])]
```

Comment:
Can you explain this change?  Is it just that the code was blatantly wrong before?